### DISCUSSION TITLE
export session

### DISCUSSION THREAD

**Discussion by rprimus** [tim-we/tabs-aside#148] [Ideas]
Mon 6 May 2024 07:46:53 BST

Add the ability to export a session (`.json`, `.html`).

Context:

"Moving" sessions between profiles (e.g., `work` -> `personal`).

**Comment by tim-we** [comment]
I think that is a valid use case but since Tabs Aside stores tabs as bookmarks maybe this could be a bit more general? Like I would expect this to be a feature of a bookmark manager extension. So I would ask you to check if there is a extension that can do that or request such a feature for one of the bookmark managers. If they don't want to do it I'm happy to give it a shot (if I find the time). Of course this would also require an import feature to go along with it.
